rediscluster/clu.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+"""
+redis cluster command line interface
+
+Copyright (c) 2017, Christian Matthaei
+"""
+
+from argparse import ArgumentParser
+from rediscluster import StrictRedisCluster
+from pprint import pprint
+import re
+import socket
+import readline
+
+
+def get_parser():
+    """Get argument parser -> ArgumentParser"""
+
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        '--node', '-n', help='hostname of any cluster node',
+        default='127.0.0.1'
+    )
+    parser.add_argument(
+        '--pw', '-a', help='pw to auth, same as for redis-cli'
+    )
+    parser.add_argument(
+        '--port', '-p', help='port where redis is running', default=7000
+    )
+
+    return parser
+
+
+def get_redis_config():
+    """Get password and port from local installed redis instance
+
+    These credentials are used, when no values are passed to the
+    ArgumentParser
+    """
+
+    with open('/etc/redis/redis.conf', 'r') as config_file:
+        for line in config_file:
+            if re.match('^requirepass(.*)', line):
+                password = line.split()[1]
+            if re.match('^port(.*)', line):
+                port = line.split()[1]
+    return port, password
+
+
+def main():
+    args = get_parser().parse_args()
+
+    redis_config = get_redis_config()
```

Comment:
This is the wrong way to handle multiple arguments sent back from a method that returns multiple args. This should be something like

```
port, password = get_redis_config()

if port:
  args.port = port
if password:
  args.pw = password
```

Reply:
changed